Description: Drop use of git from sphinx build config file as git repository
             is not available during package builds.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: no

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -57,10 +57,10 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = '%sdoc' % project
 
-git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local",
-           "-n1"]
-html_last_updated_fmt = subprocess.check_output(git_cmd,
-                                                stdin=subprocess.PIPE)
+#git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local",
+#           "-n1"]
+#html_last_updated_fmt = subprocess.check_output(git_cmd,
+#                                                stdin=subprocess.PIPE)
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass
